04-visualization-ggplot2.Rmd

```diff
@@ -598,14 +598,14 @@ library(gridExtra)
 spp_weight_boxplot <- ggplot(data = surveys_complete, 
                              mapping = aes(x = genus, y = weight)) +
   geom_boxplot() +
-  xlab("Genus") + ylab("Weight (g)") +
   scale_y_log10() +
+  labs(x = "Genus", y = "Weight (g)") +
```

Comment:
I moved this downwards, because all other `labs()` are directly above a `theme` function.